ddtrace/tracer/span.go

```diff
@@ -68,10 +70,12 @@ type span struct {
 	ParentID uint64             `msg:"parent_id"`         // identifier of the span's direct parent
 	Error    int32              `msg:"error"`             // error status of the span; 0 means no errors
 
-	noDebugStack bool         `msg:"-"` // disables debug stack traces
-	finished     bool         `msg:"-"` // true if the span has been submitted to a tracer.
-	context      *spanContext `msg:"-"` // span propagation context
-	taskEnd      func()       // ends execution tracer (runtime/trace) task, if started
+	noDebugStack   bool            `msg:"-"` // disables debug stack traces
+	finished       bool            `msg:"-"` // true if the span has been submitted to a tracer.
+	context        *spanContext    `msg:"-"` // span propagation context
+	labelContext   context.Context `msg:"-"` // context with profiler labels applied for this span
```

Comment:
```suggestion
	labelContext   context.Context `msg:"-"` // context with profiler labels as provided by pprof.WithLabel
```
Would this be better? I feel it gives a bit clearer insight into where it's from / what it is.

I would even consider calling it `pprofContext`